package.json

```diff
@@ -19,18 +19,25 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.27.2",
+    "html-react-parser": "^3.0.8",
+    "localforage": "^1.10.0",
+    "match-sorter": "^6.3.1",
     "eslint-config-react-app": "^7.0.1",
     "hamburger-react": "^2.5.0",
     "moment": "^2.29.4",
     "next": "^12.3.1",
     "react": "^17.0.2",
     "react-circular-progressbar": "^2.1.0",
     "react-dom": "^17.0.2",
+    "react-hot-toast": "^2.4.0",
```

Comment:
For the nice toast floating above when an action happens

Reply:
Mmm toast

Reply:
@gregv French toast is served! Bon appetit monsieur  🍞😂